numpy/polynomial/_polybase.py

```diff
@@ -805,6 +817,9 @@ class domain in NumPy 1.4 and ``None`` in later versions.
         if full:
             [coef, status] = res
             return cls(coef, domain=domain, window=window), status
+        elif cov:
```

Comment:
why is cov and full mutually exclusive? Shouldn't it be something like `if full and not cov` , `elif cov and not full` ....

Reply:
I suppose it is a little ambiguous. I was just copying the style of `polyfit` to keep it consistent: https://github.com/numpy/numpy/blob/d9b1e32cb8ef90d6b4a47853241db2a28146a57d/numpy/lib/polynomial.py#L639-L6419